asterius/rts/rts.heapalloc.mjs

```diff
@@ -1,13 +1,42 @@
 import * as rtsConstants from "./rts.constants.mjs";
 
+/**
+ * Class implementing the allocation of heap objects.
+ * This mainly consists of a block allocator, which allocates
```

Comment:
In the current asterius rts, we do not really distinct between mblocks and blocks, and thus all blocks are really mblocks.